missions/2PzD_Template.VR/customization/loadouts/US/loadoutUSAB42.sqf

```diff
@@ -416,3 +425,38 @@
         [US_Weap_Bazoo] call Olsen_FW_FNC_AddItem;
         [US_Mag_Bazoo,3,"backpack"] call Olsen_FW_FNC_AddItem;
     }];
+
+    //Recon Team leader
+    USAB42_RCTL = ["USAB42_RCTL", {
+        params ["_unit"];
+
+        [USAB_UniK_CPL] call Olsen_FW_FNC_AddItem;
+        [USAB_Vest_M1G] call Olsen_FW_FNC_AddItem;
+        USAB_Backpack("Rif");
+        [USAB_Helm_r] call Olsen_FW_FNC_AddItemRandom;
+        [GEN_Face_r] call Olsen_FW_FNC_AddItemRandom;
+
+        //Assigned Items
+        GEN_Default_Equipment_Set;
+        GEN_Leader_Equipment_Set;
+
+        //Primary Weapon
+        USAB42_Weapon_Rifle;
+    }];
+
+    //Recon Team Sniper
+    USAB42_RCTS = ["USAB42_RCTS", {
+        params ["_unit"];
+
+        [USAB_UniK_PFC] call Olsen_FW_FNC_AddItem;
+        [USAB_Vest_M1G] call Olsen_FW_FNC_AddItem;
+        USAB_Backpack("Rif");
```

Comment:
Could you change the parameter here to match this particular case's abbreviation? So `"USAB_Backpack("RCTS");` Same for the other USAB42 and 44 cases